jobs/config.json

```diff
@@ -2539,27 +2539,27 @@
       "UNKNOWN"
     ]
   },
-  "ci-kubernetes-e2e-gce-master-new-cvm-kubectl-skew": {
+  "ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew": {
     "args": [
       "--check-leaked-resources",
       "--check-version-skew=false",
       "--env-file=jobs/platform/gce.env",
-      "--env-file=jobs/ci-kubernetes-e2e-gce-master-new-cvm-kubectl-skew.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew.env",
       "--extract=ci/latest-1.7",
       "--extract=ci/latest",
-      "--gcp-project=gce-cvm-upg-1-5-1-4-ctl-skew",
+      "--gcp-project=gce-gci-upg-1-5-1-4-ctl-skew",
```

Comment:
now gci-skew and gci-skew-serial are sharing the same projects - you can remove both `--gcp-project` flags and add the projects to https://github.com/kubernetes/test-infra/blob/master/boskos/resources.json